libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -158,14 +158,14 @@ void AP_GPS_Backend::_detection_message(char *buffer, const uint8_t buflen) cons
 
 void AP_GPS_Backend::broadcast_gps_type() const
 {
-    char buffer[64];
```

Comment:
This is correct for the status text, but can't we log 64 bytes to the DF log if we wanted? (Since this information can conceivably come from the GPS unit in the near future I'm wondering if we want maximum fidelity in logging). I'm not to bothered either way since everything we currently use definitely fits.

Reply:
I don't think it would be appropriate to have `gcs().send_text(...)` work that way :-)

In fact, `gcs().send_text(...)` should throw a warning in SITL if it truncates, I think.